https://github.com/flightaware/tclx/commit/dfd54117ca500c9422ff2e11940219d19cedb906
https://bugs.gentoo.org/881713
https://sourceforge.net/p/tclx/bugs/84/

From dfd54117ca500c9422ff2e11940219d19cedb906 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Tue, 23 Jun 2020 22:44:30 -0700
Subject: [PATCH] Add missing #includes during configure tests

These lead to failures when configuring on macOS on Apple Silicon

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
--- a/configure.in
+++ b/configure.in
@@ -195,6 +195,7 @@ WIFEXITED(x);	/* Generates compiler error if WIFEXITED uses an int. */
     AC_TRY_RUN([
 #include <sys/types.h>
 #include <sys/times.h>
+#include <unistd.h>
 main() {
     struct tms cpu;
     times(&cpu);
--- a/tclconfig/tcl.m4
+++ b/tclconfig/tcl.m4
@@ -2572,6 +2572,7 @@ AC_DEFUN([TEA_BUGGY_STRTOD], [
     if test "$tcl_strtod" = 1; then
 	AC_CACHE_CHECK([for Solaris2.4/Tru64 strtod bugs], tcl_cv_strtod_buggy,[
 	    AC_TRY_RUN([
+		#include <stdlib.h>
 		extern double strtod();
 		int main() {
 		    char *infString="Inf", *nanString="NaN", *spaceString=" ";

